### DISCUSSION TITLE
Connecting to wallet....

### DISCUSSION THREAD

**Discussion by iamaer4fa** [Chia-Network/chia-blockchain#4900] [Support]
_Transferred to chia-blockchain GUI._

Hi,

I am on Win64 10, Ryzen CPU 7 1700, 32Gb RAM, GTX 1050Ti, 4 Plots,  Attempting to run chia, but it will never finish connecting, its been many hours waiting for it to connect. (Using Chia GUI version 1.1.4) 

_Transferred to chia-blockchain GUI._

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
